### PR TITLE
Added CentOS support

### PR DESCRIPTION
**Pull request by vladsokolovsky**

### CODE REVIEW

File: src/bfb_to_raw_img.sh [hunk 1]
```diff
@@ -94,12 +101,7 @@ fi
 
 #check that file exist and it is a bfb file
 if [ -f "$bfb" ]; then
-    if [[ $bfb == *ubuntu* ]]; then
-        log "INFO: starting creating img from $bfb"
```
Comment (ealkobi) [inline src/bfb_to_raw_img.sh:98]: maybe we should add a check to verify that bfb file name includes ubuntu or centos

File: src/bfb_to_raw_img.sh [hunk 2]
```diff
@@ -124,8 +126,13 @@ bfb_basename=`basename $bfb`
 WDIR=/tmp/$bfb_basename$id
 mkdir -p $WDIR
 
+if [ "$os" == "centos" ]; then
+    cp bfb_tool_raw_img.centos.sh $WDIR/bfb_tool_raw_img.sh
+else
+    cp bfb_tool_raw_img.sh $WDIR
```
Comment (ealkobi) [inline src/bfb_to_raw_img.sh:132]: I think we should modify bfb_tool_raw_img.sh to be called bfb_tool_raw_img.ubuntu.sh to be consistent.

File: src/bfb_tool_raw_img.centos.sh [hunk 3]
```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+
```
Comment (ealkobi) [inline src/bfb_tool_raw_img.centos.sh:2]: Can you please explain what was missing from the last time we tried to create a VM image based on Centos BFB?

File: src/bfb_to_raw_img.sh (hunk 2 above)
Comment (ealkobi) [inline src/bfb_to_raw_img.sh:132]: I think we should modify bfb_tool_raw_img.sh to be called bfb_tool_raw_img.ubuntu.sh to be consistent.

File: src/bfb_to_raw_img.sh [hunk 4]
```diff
@@ -94,12 +101,7 @@ fi
 
 #check that file exist and it is a bfb file
 if [ -f "$bfb" ]; then
-    if [[ $bfb == *ubuntu* ]]; then
```
Comment (ealkobi) [inline src/bfb_to_raw_img.sh:97]: maybe we should add a check to verify that bfb file name includes ubuntu or centos

File: src/bfb_to_raw_img.sh [hunk 5]
```diff
@@ -65,6 +67,11 @@ while [[ $# -gt 0 ]]; do
             shift # past argument
             shift # past value
             ;;
+        -os|--os)
```
Comment (ealkobi) [inline src/bfb_to_raw_img.sh:70]: Maybe instead of adding an argument we can identify from bfb file the OS and running the relevant script inside docker.

**Comment by vladsokolovsky** [comment]
Hi Emanuel,
My commit adds a new functionality (CentOS BFB support) without changing existing environment.
If you want to do further changes it is up to you.
In general, you need to maintain bfb_tool_raw_img script per OS as it includes OS specific stuff (partitioning, file system type, configuration, …).

Regards,
Vladimir

**Pull request merged**